pyproject.toml

```diff
@@ -50,11 +50,11 @@ isort = "*"
 pygments = "*"
 pytest = "^7.0.0"
 pytest-pycharm = "*"
-# Ensure the Sphinx version remains synchronized with
-# docs/requirements.txt to maintain consistent output during both
-# development and publishing on Read The Docs.
-sphinx = "~=4.4.0"
-sphinx-rtd-theme = "1.3.0"
+# Ensure the Sphinx version remains synchronized with docs/requirements.txt
+# to maintain consistent output during both development and publishing on
+# Read The Docs.
+sphinx = "^7.1.0"
```

Comment:
7.2.0 drops support for Python 3.8. Building the docs only really needs the latest version, but there's no difference to us functionally between 7.1.0 and later versions for now.